apps/documents/assets/ParagraphForm.jsx

```diff
@@ -82,12 +97,19 @@ class ParagraphForm extends React.Component {
             </div>
 
             <div className="form-group">
-              <label
-                htmlFor={'id_paragraphs-' + this.props.id + '-text'}
-              >
-                {django.gettext('Paragraph')}
+              <label htmlFor={'id_paragraphs-' + props.id + '-text'}>
+                {translations.paragraph}
+                <div
+                  id={'id_paragraph-help-text-' + props.id}
+                  className="form-hint"
+                >
+                  {translations.helpText}
+                </div>
                 <div
                   className="django-ckeditor-widget"
+                <div
```

Comment:
something is wrong here `<div [...] div<`

Reply:
oh totally... i updated it.